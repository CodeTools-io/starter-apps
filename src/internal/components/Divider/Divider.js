import React from 'react';
import { Box, Text } from 'grommet';

export default function Divider({ text, ...props }) {
  return (
    <Box
      pad="medium"
      direction="row"
      justify="center"
      align="center"
      gap="medium"
      width="33%"
      flex={false}
    >
      <Box border="bottom" width="xsmall" />
      <Text size="xxlarge" weight="bold">
        {text}
      </Text>
      <Box border="bottom" width="xsmall" />
    </Box>
  );
}
