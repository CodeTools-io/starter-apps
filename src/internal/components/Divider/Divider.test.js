import React from 'react'
import Divider from './Divider'

describe('Divider', () => {
  test('can render', () => {
    const { container } = renderComponent(
      <Divider>
        <p>Content for the Divider</p>
      </Divider>
    )

    expect(container).toMatchSnapshot()
  })
})

