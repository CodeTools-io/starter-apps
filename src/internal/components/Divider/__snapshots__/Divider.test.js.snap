// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider can render 1`] = `
<div>
  <div
    class="StyledBox-sc-13pk1d4-0 liGFHN"
  >
    <div
      class="StyledBox-sc-13pk1d4-0 izWPUw"
    />
    <div
      class="StyledBox__StyledBoxGap-sc-13pk1d4-1 ffHQai"
    />
    <span
      class="StyledText-sc-1sadyjn-0 eTkzRk"
    />
    <div
      class="StyledBox__StyledBoxGap-sc-13pk1d4-1 ffHQai"
    />
    <div
      class="StyledBox-sc-13pk1d4-0 izWPUw"
    />
  </div>
</div>
`;
